public/_redirects

```diff
@@ -1,23 +1,23 @@
-/feedback        https://forms.gle/V9hjVGn5kBzjfWFu7                                                                                               301
-/facebook        https://facebook.com/ucfsigai                                                                                                     301
-/instagram       https://www.instagram.com/_ucfai                                                                                                  301
-/kaggle          https://kaggle.com/ucfaibot                                                                                                       301
-/newsletter      https://us19.campaign-archive.com/home/?u=f1da4378398d0f3af136bfb7f&id=5a2c1d8a24                                                 301
-/linktree        https://linktr.ee/ucfai                                                                                                           301
-/apply           https://forms.gle/WZJv9ttNH3W9pQtQ9                                                                                               301
-/documentation   https://www.notion.so/ucfai/481486ee6ef84391bacfb68a760f9aff?v=13c996434d4143c689726b4e1417ba2a                                   301
-/notion          https://www.notion.so/ucfai/481486ee6ef84391bacfb68a760f9aff?v=13c996434d4143c689726b4e1417ba2a                                   301
-/youtube         https://www.youtube.com/channel/UCyokaec6yvsotHXFz9pbcaA                                                                          301
-/drive           https://drive.google.com/drive/folders/1vGyy2p8m7iUXxDK2ii8ZH2HUjoy6BRBp?usp=sharing                                              301
-/knightconnect   https://knightconnect.campuslabs.com/engage/organization/ucfai                                                                    301
-/promo-slide     https://docs.google.com/presentation/d/1O11hwpTto5Dp6VTSIPqoMDo00Aw-NX5FuFs1qevOXkk                                               301
-/promo-image     https://drive.google.com/file/d/1tDY81lTwhOyqZClN-hjb-T4nH61uMxV4/view?usp=share_link                                             301
-/flyer           https://drive.google.com/file/d/1X9QU3IUqPyNoS9a6dZsLHCCkpAzfPF1g/view?usp=sharing                                                301
-/poster          https://drive.google.com/file/d/1X9QU3IUqPyNoS9a6dZsLHCCkpAzfPF1g/view?usp=sharing                                                301
-/shirt           https://docs.google.com/forms/d/e/1FAIpQLSe9hl_fMxxsFskmVWDFXgag2v7vLiXbqlwTveArEpuw8oFz6w/viewform                               301
+/discord         https://discord.gg/7vwjC8P                                                                                                                                              301
+/admin-calendar  https://calendar.google.com/calendar/u/0?cid=M2E1MDQ3ZjUxNGRlYmY3M2EzM2ZhN2Y3NzM1NGVhZTFlZDlhNGJjZTlmZGNmOTkyOWY2YWE1MzY4ZWQ1MmU5NkBncm91cC5jYWxlbmRhci5nb29nbGUuY29t   301
```

Comment:
this looks like it's including google's `/u/N` string. i don't know that this will correctly work. can you remove that part of the string?